src/app/Main/Body/QueryPage/NumberOfIsolates/QueryPage-NrOfIsolates.component.tsx

```diff
@@ -2,18 +2,9 @@
 import { css, jsx } from "@emotion/core";
 import { useTranslation } from "react-i18next";
 import { useContext } from "react";
-import Accordion from "@material-ui/core/Accordion";
-import AccordionSummary from "@material-ui/core/AccordionSummary";
-import AccordionDetails from "@material-ui/core/AccordionDetails";
-import ExpandMoreIcon from "@material-ui/icons/ExpandMore";
 import { DataContext } from "../../../../Shared/Context/DataContext";
+import { AccordionComponent as Accordion } from "../../../../Shared/Accordion.component";

```

Comment:
I'm not sure about how good it is to rename imported things in the import statement. I think it is only useful if you import things with conflicting names. I prefer to remove those renames.
This mostly happens if you rename something with vscode from a file where it is imported and not from the file where it is defined. So be sure to rename in the file where the object is defined

Reply:
I just removed the "component" from the name, because it seems so long and useless inside the code. But yes, I can remove it.

Reply:
As you like. The name is local to your file so you can name it as you like. But if you use vs-code auto-import the full name is imported anyway and then you have mixed naming patterns (if you not want to manually change names after import)

Reply:
Okay. I got the point ;). Removed it in the commit "Remove aliases inside import"